examples/resources/coder_parameter/resource.tf

```diff
@@ -74,4 +74,10 @@ data "coder_parameter" "cat_lives" {
 data "coder_parameter" "fairy_tale" {
   name = "Fairy Tale"
   type = "string"
+}
+
+data "coder_parameter" "users" {
+  name    = "System users"
+  type    = "list(string)"
+  default = jsonencode(["root", "user1", "user2"])
```

Comment:
This is really unfortunate, but we need to pick a single type of property. We need to fallback to string.

Reply:
I think it is ok just wondering if `list:string` could be better to parse if we parse it at all. Just a thought, the current one looks good too.

Reply:
It would be definitely easier to parse, but it is aligned with Terraform now, so I'd rather keep like this.